staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2466,7 +2466,7 @@ type Taint struct {
 	// TimeAdded represents the time at which the taint was added.
 	// It is only written for NoExecute taints.
 	// +optional
-	TimeAdded metav1.Time `json:"timeAdded,omitempty" protobuf:"bytes,4,opt,name=timeAdded"`
+	TimeAdded *metav1.Time `json:"timeAdded,omitempty" protobuf:"bytes,4,opt,name=timeAdded"`
```

Comment:
Isn't it a breaking API change in beta type?

Reply:
wire serialization remains compatible.

the old version encoded zero values to:
*  json: `timeAdded: null`
*  proto: 0x22 tag with zero length value

the new version omits zero values from the wire format

the new version successfully decodes zero values from the old version

the old version treats missing values as the zero value when decoding